src/odemis/acq/fastem.py

```diff
@@ -320,12 +320,22 @@ def run(self):
         # Update the position of the first tile.
         self._pos_first_tile = self.get_pos_first_tile()
 
-        # Move the stage to the first tile, to ensure the autofocus and image translation pre-align are
-        # done in the correct location and the correct position is stored in the megafield metadata yaml file.
-        self.move_stage_to_next_tile()
-
         if self._pre_calibrate:
+            # The pre-calibrations should be done a full field outside the ROA,
+            # therefore temporarily set the overlap to zero.
+            overlap_init = self._roa.overlap
+            self._roa.overlap = 0
```

Comment:
This is a bit ugly, but I don't have a better idea of solving this.

Reply:
mhm I also do not have a better idea right now.